src/components/Score.js

```diff
@@ -3,16 +3,40 @@ import { browserHistory } from 'react-router';
 
 import ScoreInput from './ScoreInput.js';
 
+const pushPayload = {
+  sessionId: null,
+  scopes: {}
+};
+
 const Score = React.createClass({
+  updatePushPayload(scope, score, comment) {
+    pushPayload.sessionId = this.props.params.sessionId;
+    pushPayload.scopes[scope] = pushPayload.scopes[scope] || {};
+    pushPayload.scopes[scope].score = score;
+    pushPayload.scopes[scope].comment = comment;
+  },
   submitScores() {
-    browserHistory.push('/d/' + this.props.params.sessionId);
+    // TODO: validation - pk2
+    fetch('http://localhost:3000/api/score', {
+      method: 'post',
+      headers: {
+        'accept': 'application/json',
+        'content-type': 'application/json'
+      },
+      body: JSON.stringify(pushPayload)
+    })
+    .then(() => {
+      browserHistory.push('/d/' + this.props.params.sessionId);
+    }).catch(() => {
+      // console.log(resp, 'a bad thing happened');
+    });
```

Comment:
`console.error` or barfing something to Senrty seems good and just here. But doing something on error seems like another issue we can file.